### ISSUE TITLE
Connection drops during multiple streams

### ISSUE THREAD

**Issue by KarthikaKumar** [vzhn/bstreamer#14]
Hi @vzhn  we are trying to run 64 streams, basically 64 connections have been made by the clients. But when we leave the bstreamer simulator for longer run, after few hrs, we are seeing recording gaps observed on the client, and on bstreamer lag count is increasing. Any reason this could happen or do you have any suggestions please ?

**Comment by vzhn** [comment]
Hi! When lag count is increased it means that server is trying to send next chunk of video, but not all clients received current chunk of video. When server could not send chunk of video, it just skip it. Maybe client is slow, maybe it is network problems, maybe recorder is not fast enough to record video, and so on.

**Comment by KarthikaKumar** [comment]
Thanks for your response. The problem here is it works with the real cameras without any recording gaps. How do we get the root cause? right now we have plans to move the simulator to a physical machine instead of VM and also reduced the load from 64 to 32.

**Comment by vzhn** [comment]
Hmm, what is VM? Is it Windows VM? I have similar problems with Window in the past.
Also you can increase following setting values:
`watermarks.high: `
`limits.size: `

**Comment by KarthikaKumar** [comment]
Yes it is windows VM. Ok I will try increasing those values.Thanks for
helping

**Comment by vzhn** [comment]
Oh, Windows is terrible for some reason, don't know why.

**Comment by KarthikaKumar** [comment]
Ok will try it in Linux as well then, thanks again
